### PR TITLE
added combat formulas

### PR DESCRIPTION
**Pull request by hojerst**
Added combat formulas based on old formulas found on an archived version of wiki.eve-id.net.

### CODE REVIEW

File: docs/formulas/combat.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Combat
+
+## Target lock time
+
+The target lock time (`$ t_{targetlock} $`) in seconds depends on the ship's scan resolution (`$ s $`) and the target's signature radius (`$ r $`)
+
+```math
+t_{targetlock} := {40000 \over s \cdot asinh(r)^2}
```
Comment (Regner) [inline docs/formulas/combat.md]: Whats with the colon in front of the equals sign?

I am new to the match stuff but I don't see it in either of https://recommonmark.readthedocs.org/en/latest/auto_structify.html#math-formula or https://recommonmark.readthedocs.org/en/latest/auto_structify.html#inline-math

Comment (hojerst) [inline docs/formulas/combat.md]: I learned ':=' as "defined as". However some people just use '=' for the same meaning. I can remove the : if you prefer that. (later today)

**Comment by Regner** [comment]
Looks good to me. Will integrate and @jimc can point out anything else that needs to change but that is all I see. >.<

Thanks!

**Comment by JimiC** [comment]
@Regner Looked good to me, that's why I added the :+1:

**Pull request merged**